src/diceBot/AFF2e.rb

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+class AFF2e < DiceBot
+  # ダイスボットで使用するコマンドを配列で列挙する
+  setPrefixes(['FF.+', 'FR.+', 'FD.+'])
+
+  def gameName
+    # 'ADVANCED FIGHTING FANTASY 2nd Edition'
+    'AFF2e'
+  end
+
+  def gameType
+    'AFF2e'
+  end
+
+  def getHelpMessage
+    return <<MESSAGETEXT
+対抗なしロール\tFF{目標値}+{補正}
+対抗ロール\tFR{能力値}+{補正}
+武器ロール\tFD[2,3,3,3,3,3,4]+{補正}
+防具ロール\tFD[0,0,0,0,1+1,1+1,2+2]+{補正}
+MESSAGETEXT
+  end
+
+  def explicit_sign(i)
+    format('%+d', i)
+  end
+
+  def eval_term(term)
+    value = 0
+    term.scan(/[+-]?\d+/) do |fact|
+      value += fact.to_i
+    end
+    value
+  end
+
+  def parentheses(str)
+    '(' + str + ')'
+  end
+
+  def successful_or_failed(total, diff)
+    case total
+    when  2
+      diff <=  1 ? '成功（大成功ではない）' : '大成功！'
+    when 12
+      diff >= 12 ? '失敗（大失敗ではない）' : '大失敗！'
+    else
+      total <= diff ? '成功' : '失敗'
+    end
+  end
+
+  def clamp(i, min, max)
+    i = min if i < min
+    i = max if i > max
+    i
```

Comment:
```suggestion
    if i < min
      min
    elsif i > max
      max
    else
      i
    end
```
`min <= max` の前提があるなら上記で良いと思うのですがどうでしょう

Reply:
そうですね。
Comparable#clamp を使いたいところです。Ruby 2.4 以降